scRNA-seq-advanced/04-differential_expression.Rmd

```diff
@@ -336,125 +346,162 @@ counts_mtx
 
-Now we are ready to set up our `DESeq2` object. 
+Now we will use the `DESeq2` package to perform DE analysis on our pseudo-bulked SCE object.
+From this point, we can proceed in the same way we would if we had a bulk RNA-seq dataset with 6 samples.
+We will start with the unnormalized raw counts in the `counts` assay of the pseudo-bulked SCE and do the following with `DESeq2`:
 
-```{r}
-# deseq2 requires the coldata object as a separate input
-#  in data.frame format
+- Create a `DESeq` object
+- Normalize and log transform the counts data
+- Estimate dispersions and shrink estimates
+- Fit a negative binomial model and perform hypothesis testing using Wald statistics
+
+#### Create the `DESeq2` Object
+
+First we will create the `DESeq2` object.
+To create the object we will need the unnormalized counts matrix, the metadata associated with the samples that will help tell `DESeq2` which sample belongs to which group, and the design formula. 
+The design formula is used to indicate which columns of the metadata need to be considered in the DE comparison.
+For our experiment we are comparing gene expression between different RMS subtypes, stored in the `diagnosis_group` column of the `colData` in the pseudo-bulked SCE.
+
+```{r deseq object}
+# extract the metadata for each sample from sce as data.frame
 coldata_df <- as.data.frame(colData(tumor_myoblast_sce))
   
 # set up the deseq object 
 deseq_object <- DESeq2::DESeqDataSetFromMatrix(counts(tumor_myoblast_sce),
+                                               # metadata from coldata 
                                                colData = coldata_df,
+                                               # name of column to use for grouping samples 
                                                design = ~ diagnosis_group)
 ```
 
-Use the median of ratios method for count normalization followed by regularized log transformation.
+Before we can compare gene expression we will need to normalize the counts data.
+If you look at the pseudo-bulked SCE object you should have only seen one assay, the `counts` assay.
+This is because we took the unnormalized counts from the original SCE to create the new pseudo-bulked counts which then become the input to create the `DESeq2` object.
+We don't want to sum the normalized counts because those have each been normalized within their sample and do not account for differences between samples.
+When we run `DESeq2` on our dataset, raw counts will first be normalized using size factors to account for differences in total sample counts.
+Therefore we don't have to do any normalization on our own and we can let `DESeq2` do all the work for us.
 
-```{r}
-# estimate size factors first 
+However, we want to take a look at our data before we run the DE analysis.
+In particular, we can use principal component analysis (PCA) of our pseudo-bulked dataset as an initial test of our hypothesis that ARMS and ERMS samples have different gene expression patterns.
```

Comment:
I'm a little unsure about calling this a test...maybe "exploration of our hypothesis"?


```suggestion
In particular, we can use principal component analysis (PCA) of our pseudo-bulked dataset as an initial exploration of our hypothesis that ARMS and ERMS samples have different gene expression patterns.
```